core/modules/filter/js/filter.js

```diff
@@ -238,12 +238,18 @@ Backdrop.behaviors.editorImageDialog = {
               var $submit = $form.find('.form-actions input[type=submit]:first');
               $submit.trigger('mousedown').trigger('click').trigger('mouseup');
             });
+          // Use autocomplete field as exposed filter for image library.
+          $('#edit-attributes-src').keyup(function () {
```

Comment:
It would be good to use `Backdrop.debounce` on this to prevent race conditions. Right now an ajax request is fired for every keyup, which might be unnecessary if you're a fast typer.